contrib/cirrus/check_image.sh

```diff
@@ -36,4 +36,28 @@ do
         "$(systemctl list-unit-files --no-legend $REQ_UNIT)" = "$REQ_UNIT enabled" || let "RET+=1"
 done
 
+# Previously, perdiodic package updates were enabled and would trigger
+# soon after an Ubuntu VM booted, grabbing a specific lock file for
+# several minutes after setup_environment.sh ran (previous script here also).
+DPKGLCK=/var/lib/dpkg/lock
+if [[ "$OS_RELEASE_ID" == "ubuntu" ]] && [[ -r "$DPKGLCK" ]]
+then
+    item_test "The lock file $DPKGLCK exists" -r "$DPKGLCK" || let "RET+=1"
+    flock --exclusive --nonblock --no-fork $DPKGLCK sleep 1h &
+    sleep 1s  # just in case
```

Comment:
Just in case what?

Reply:
Line 46 is backgrounded, I'm not sure if that means `flock` could race with `lsof` or not.  Probably very unlikely, but I didn't want to leave it to chance.  I can update comment to be more helpful :smile: